test_scripts/API/VehicleData/StabilityControlsStatus/004_OnVehicleData_StabilityControlsStatus.lua

```diff
@@ -0,0 +1,114 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: Check receiving StabilityControlsStatus data via OnVehicleData notification
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
```

Comment:
```suggestion
-- 2) App1 is registered
-- 3) App2 is registered 
```
Test case precondition does not reflect completely test script scenario. Please correct test script precondition.

Reply:
@VjKlepikov Fixed (see a7a20fe)